mo_gymnasium/envs/deep_sea_treasure/deep_sea_treasure.py

```diff
@@ -25,6 +25,19 @@
     ]
 )
 
+FRONT_DEFAULT = [
+    np.array([0.7, -1]),
+    np.array([8.2, -3]),
+    np.array([11.5, -5]),
+    np.array([14.0, -7]),
+    np.array([15.1, -8]),
+    np.array([16.1, -9]),
+    np.array([19.6, -13]),
+    np.array([20.3, -14]),
+    np.array([22.4, -17]),
+    np.array([23.7, -19]),
+]
+
```

Comment:
I don't you if I find better that these are global variables, or if they were returned by env.get_front() or something like that. What do you think?